config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/PolicyEntityBuilder.java

```diff
@@ -421,7 +421,8 @@ Entity buildPolicyEntity(Policy policy, AnnotatedBundle annotatedBundle, Bundle
         resourceSetElement.appendChild(resourceElement);
         resourcesElement.appendChild(resourceSetElement);
         policyElement.appendChild(resourcesElement);
-        Entity entity = EntityBuilderHelper.getEntityWithPathMapping(EntityTypes.POLICY_TYPE, policyNameWithPath, policy.getId(), policyElement);
+        Entity entity = EntityBuilderHelper.getEntityWithPathMapping(EntityTypes.POLICY_TYPE,
```

Comment:
Are we doing similar change in EncassEntityBuilder ?

Reply:
@uppoju Yes.